nvflare/app_opt/xgboost/histogram_based/executor.py

```diff
@@ -73,15 +73,15 @@ class FedXGBHistogramExecutor(Executor):
     This class implements a basic xgb_train logic, feel free to overwrite the function for custom behavior.
     """
 
-    def __init__(self, num_rounds, early_stopping_round, xgboost_params: dict, data_loader_id: str, verbose_eval=False):
+    def __init__(self, num_rounds, early_stopping_rounds, xgb_params: dict, data_loader_id: str, verbose_eval=False):
         """Federated XGBoost Executor for histogram-base collaboration.
 
         This class sets up the training environment for Federated XGBoost.
         This is the executor running on each NVFlare client, which starts XGBoost training.
 
         Args:
             num_rounds: number of boosting rounds
-            early_stopping_round: early stopping round
+            early_stopping_rounds: early stopping rounds
             xgboost_params: This dict is passed to `xgboost.train()` as the first argument `params`.
```

Comment:
If change this arg name from `xgboost_params` to `xgb_params`.
We also need to change all docstrings and all documentation.

For example here the docstring.